examples/playbooks/rule-args-module-fail.yml

```diff
@@ -34,3 +34,8 @@
       ansible.builtin.file:
         path: /opt/software/deployment
         state: away
+
+    - name: Create software directory (Windows module - Bug 3200) # we display a warning
```

Comment:
Move this into `-pass` file as we expect it to pass w/o any other side-effects, not to fail.